src/libguac/client.c

```diff
@@ -478,6 +478,41 @@ int guac_client_load_plugin(guac_client* client, const char* protocol) {
 
 }
 
+/**
+ * A callback function which is invoked by guac_client_owner_send_required() to
+ * send the required parameters to the specified user, who is the owner of the
+ * client session.
+ * 
+ * @param user
+ *     The guac_user that will receive the required parameters, who is the owner
+ *     of the client.
+ * 
+ * @param data
+ *     A pointer to a NULL-terminated array of required parameters that will be
+ *     passed on to the owner to continue the connection.
+ * 
+ * @return
+ *     The integer status of the operation, cast as void*.
+ */
+static void* guac_client_owner_send_required_callback(guac_user* user, void* data) {
+    
+    const char** required = (const char **) data;
+    
+    /* Send required parameters to owner. */
+    return (void*) ((intptr_t) guac_protocol_send_required(user->socket, required));
```

Comment:
Beware that `user` can be `NULL` if the owner has left the connection. See:

https://guacamole.apache.org/doc/1.1.0/libguac/client_8h.html#af3f4ed85d98b16376e2cdc031ff1b44a